src/app/static/src/app/store/adrUpload/actions.ts

```diff
@@ -91,7 +90,7 @@ export const actions: ActionTree<ADRUploadState, RootState> & ADRUploadActions =
                 break
             }
         }
-        await dispatch("adr/getAndSetDatasets", selectedDatasetId);
-        dispatch("getUploadFiles");
+        await dispatch("adr/getAndSetDatasets", selectedDatasetId, {root: true});
```

Comment:
Could add a test for the new `{root: true}` parameter.

Reply:
Done